pkg/testers/ginkgo/package.go

```diff
@@ -72,7 +73,12 @@ func (t *Tester) AcquireTestPackage() error {
 }
 
 func (t *Tester) extractBinaries(downloadPath string) error {
-	// finally, search for the test package and extract it
+	// ensure the artifacts dir
+	if err := os.MkdirAll(artifacts.BaseDir(), os.ModePerm); err != nil {
```

Comment:
I think we already do this as part of the main kubetest2 app code.

Reply:
Yes, but kubetest2-test-kops calls this directly